css/styles.css

```diff
@@ -146,3 +159,141 @@ li > p {
 .vector {
   height: 30px;
 }
+
+/* Mobile version */
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+  background: #191765;
+}
+
+.grid-item {
+  align-self: auto;
+  text-align: center;
+}
+
+.element {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 0;
+}
+
+.work {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #a2a3ae;
+}
+
+.recent-work {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #9490ff;
+}
```

Comment:
- [ ] Kindly change the paragraph color to be like the Figma design.

![image](https://user-images.githubusercontent.com/35535130/154597467-3c608bf9-dc40-45cf-8558-2e3b3e36f4fa.png)